[2023-09-12 14:16:47,946] INFO _log() L224   * Restarting with stat - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:16:55,139] INFO _log() L224  127.0.0.1 - - [12/Sep/2023 14:16:55] "POST /cliente HTTP/1.1" 200 - - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:16:56,437] INFO _log() L224  127.0.0.1 - - [12/Sep/2023 14:16:56] "GET /clientes HTTP/1.1" 200 - - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:18:30,326] INFO _log() L224   * Detected change in 'C:\\Users\\jpmar\\OneDrive\\Documentos\\Projetos\\Visual Studio 2019\\Puc\\SitePuc2\\Back\\schemas\\cliente.py', reloading - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:18:30,512] INFO _log() L224   * Restarting with stat - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:19:27,777] INFO _log() L224   * Detected change in 'C:\\Users\\jpmar\\OneDrive\\Documentos\\Projetos\\Visual Studio 2019\\Puc\\SitePuc2\\Back\\schemas\\cliente.py', reloading - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:19:27,979] INFO _log() L224   * Restarting with stat - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:19:46,545] WARNING add_cliente() L320  Erro ao adicionar cliente 'afa', Cliente de mesmo nome já salvo na base :/ - call_trace=C:\Users\jpmar\OneDrive\Documentos\Projetos\Visual Studio 2019\Puc\SitePuc2\Back\app.py L320 
[2023-09-12 14:19:46,548] INFO _log() L224  127.0.0.1 - - [12/Sep/2023 14:19:46] "[31m[1mPOST /cliente HTTP/1.1[0m" 409 - - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:19:47,494] INFO _log() L224  127.0.0.1 - - [12/Sep/2023 14:19:47] "GET /clientes HTTP/1.1" 200 - - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:20:08,067] INFO _log() L224   * Detected change in 'C:\\Users\\jpmar\\OneDrive\\Documentos\\Projetos\\Visual Studio 2019\\Puc\\SitePuc2\\Back\\schemas\\cliente.py', reloading - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:20:08,237] INFO _log() L224   * Restarting with stat - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:20:09,859] INFO _log() L224   * Detected change in 'C:\\Users\\jpmar\\OneDrive\\Documentos\\Projetos\\Visual Studio 2019\\Puc\\SitePuc2\\Back\\app.py', reloading - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:20:10,027] INFO _log() L224   * Restarting with stat - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:20:17,151] INFO _log() L224  127.0.0.1 - - [12/Sep/2023 14:20:17] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:20:17,401] INFO _log() L224  127.0.0.1 - - [12/Sep/2023 14:20:17] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:20:17,405] INFO _log() L224  127.0.0.1 - - [12/Sep/2023 14:20:17] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:20:17,407] INFO _log() L224  127.0.0.1 - - [12/Sep/2023 14:20:17] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:20:17,669] INFO _log() L224  127.0.0.1 - - [12/Sep/2023 14:20:17] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:20:17,682] INFO _log() L224  127.0.0.1 - - [12/Sep/2023 14:20:17] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:20:19,482] INFO _log() L224  127.0.0.1 - - [12/Sep/2023 14:20:19] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:20:19,519] INFO _log() L224  127.0.0.1 - - [12/Sep/2023 14:20:19] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:20:19,520] INFO _log() L224  127.0.0.1 - - [12/Sep/2023 14:20:19] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:20:19,524] INFO _log() L224  127.0.0.1 - - [12/Sep/2023 14:20:19] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:20:19,679] INFO _log() L224  127.0.0.1 - - [12/Sep/2023 14:20:19] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:20:19,683] INFO _log() L224  127.0.0.1 - - [12/Sep/2023 14:20:19] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:20:20,552] INFO _log() L224  127.0.0.1 - - [12/Sep/2023 14:20:20] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:20:20,578] INFO _log() L224  127.0.0.1 - - [12/Sep/2023 14:20:20] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:20:20,583] INFO _log() L224  127.0.0.1 - - [12/Sep/2023 14:20:20] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:20:20,585] INFO _log() L224  127.0.0.1 - - [12/Sep/2023 14:20:20] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:20:20,668] INFO _log() L224  127.0.0.1 - - [12/Sep/2023 14:20:20] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:20:20,670] INFO _log() L224  127.0.0.1 - - [12/Sep/2023 14:20:20] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:20:30,734] WARNING add_cliente() L320  Erro ao adicionar cliente 'te', Cliente de mesmo nome já salvo na base :/ - call_trace=C:\Users\jpmar\OneDrive\Documentos\Projetos\Visual Studio 2019\Puc\SitePuc2\Back\app.py L320 
[2023-09-12 14:20:30,737] INFO _log() L224  127.0.0.1 - - [12/Sep/2023 14:20:30] "[31m[1mPOST /cliente HTTP/1.1[0m" 409 - - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:20:31,886] INFO _log() L224  127.0.0.1 - - [12/Sep/2023 14:20:31] "GET /clientes HTTP/1.1" 200 - - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:21:38,398] INFO _log() L224   * Detected change in 'C:\\Users\\jpmar\\OneDrive\\Documentos\\Projetos\\Visual Studio 2019\\Puc\\SitePuc2\\Back\\schemas\\cliente.py', reloading - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:21:38,551] INFO _log() L224   * Restarting with stat - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:21:46,600] WARNING add_cliente() L320  Erro ao adicionar cliente 'af', Cliente de mesmo nome já salvo na base :/ - call_trace=C:\Users\jpmar\OneDrive\Documentos\Projetos\Visual Studio 2019\Puc\SitePuc2\Back\app.py L320 
[2023-09-12 14:21:46,603] INFO _log() L224  127.0.0.1 - - [12/Sep/2023 14:21:46] "[31m[1mPOST /cliente HTTP/1.1[0m" 409 - - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:21:47,588] INFO _log() L224  127.0.0.1 - - [12/Sep/2023 14:21:47] "GET /clientes HTTP/1.1" 200 - - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-09-12 14:21:55,121] INFO _log() L224   * Restarting with stat - call_trace=C:\Users\jpmar\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
